content/acra/configuring-maintaining/general-configuration/acra-addzone.md

```diff
@@ -76,18 +76,21 @@ weight: 10
 
   Path to additional CA certificate for Redis' certificate validation.
   Empty by default.
+  If not specified, acra-addzone uses value from `--tls_ca` flag.
 
 
 * `--redis_tls_client_cert=<filename>`
 
-  Path to AcraServer TLS certificate presented to Redis (AcraServer works as "client" when communicating with Redis).
+  Path to server TLS certificate presented to Redis.
```

Comment:
okay, which server? who is a server in this case?

Reply:
removed server everywhere because it is clear without it that certificate presented to XXX. To not specify is it acra-server, acra-keymaker or something else.